### DISCUSSION TITLE
nuxt dev doesn't provide an exposed IP anymore?

### DISCUSSION THREAD

**Discussion by clcoco** [nuxt/nuxt#23266] [Questions]
Hello, everyone!

After updating to the latest version (3.7.3), I noticed a change in the behavior of my development environment. Previously, it would automatically provide an IP address instead of the usual 'localhost.' However, in the latest version, it seems that I now need to use the `--host` flag to obtain an IP address. Is this change intentional?

**Comment by xjccc** [comment]
some change of this

https://github.com/nuxt/cli/pull/105
